manage/knowledge-base.rst

```diff
@@ -197,3 +197,32 @@ Delete
 Use this panel to delete the knowledge base.
 If you wish to unpublish it without deleting all of its content,
 simply disable it via the toggle button at the top of the window instead.
+
+Permissions
+-----------
+
+The general access to the knowledge base is managed via
+``knowledge_base.reader`` and ``knowledge_base.editor`` permissions. You can
+find more information about roles and permissions
+:doc:`here <roles/index>`.
+
+If you want to grant more granular permissions than to global read/edit
+permissions (e.g. edit permissions only for specific parts of the knowledge
+base), you can do so by following the steps below:
+
+- First create the base structure of the knowledge base (if not already done)
+- Create or choose one or more roles which should have granular permissions
+- Grant the ``knowlege_base.reader`` permission to the desired role(s)
+- Edit the permissions of each (sub-)category of the knowledge base
+- Assign the role to one or more users who should be able to edit the knowledge
+  base (partially)
+
+.. note::
+
+   - The permissions of a parent category are inherited to sub-categories.
+   - The permissions of a category tree can only be widened, not restricted.
```

Comment:
This may be confusing. Reader permission can be both widened to editor and restricted to none. While editor permission cannot be up/down graded.